jenkins/bootstrap.py

```diff
@@ -478,6 +478,15 @@ def find_version(call):
         ]
         return call(cmd, output=True).strip()
 
+    # Make up some tag instead of unknown
+    cmd = ['git', 'describe', '--always', '--tags', '--abbrev=14']
```

Comment:
if you want to always get a commit sha1, you probably want to use `git rev-parse --short=14 HEAD^{commit}` instead.

`git describe --always --tags` will use tags if they exist, e.g. on contrib right now:
```
$ git describe --always --tags --abbrev=14
cluster-autoscaler-0.4.0-470-g618f52dad58b52
```